Video_Report/video_clip_report.py

```diff
@@ -0,0 +1,256 @@
+import requests
+from datetime import datetime, timedelta
+import time
+import json
+import calendar
+import csv
+import sys
+import os
+import argparse
+import urllib3
+import math
+from typing import Dict, Set
+
+#to disable warnings for not verifying host
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+class faceVideo:
+    processed: Dict[str, Set]
+
+    #initializes the argument parser and the headers for media and the api sessions
+    def __init__(self, cli_args):
+        arg_parser = self.__initalize_argument_parser()
+        self.args = arg_parser.parse_args(cli_args)
+        self.api_url = "https://api2.rhombussystems.com"
+        self.api_sess = requests.session()
+        #api headers
+        self.api_sess.headers = {
+            "Accept": "application/json",
+            "x-auth-scheme": "api-token",
+            "Content-Type": "application/json",
+            "x-auth-apikey": self.args.APIkey}
+
+        self.media_sess = requests.session()
+        #media headers
+        self.media_sess.headers = {
+            "Accept": "application/json",
+            "x-auth-scheme": "api-token",
+            "Content-Type": "application/json",
+            "x-auth-apikey": self.args.APIkey}
+        #The three sets to make sure the similar events are not added to the csv
+        self.processed = dict()
+
+    #The argument parser
+    @staticmethod
+    def __initalize_argument_parser():
+        parser = argparse.ArgumentParser(
+            description= "Takes a clip from the specified time for a specified duration.")
+        #aruements avaiable for the user to customize
+        parser.add_argument('APIkey', type=str, help='Get this from your console')
+        parser.add_argument('cameraNames', type=str, help='Name of the camera or cameras in the console you want the clips from')
```

Comment:
the description needs to tell how to enter multiple camera names.  for example, a comma separated list of camera names